runtime/vm/CMakeLists.txt

```diff
@@ -124,6 +127,23 @@ add_library(j9vm SHARED
 	${CMAKE_CURRENT_BINARY_DIR}/ut_j9vm.c
 )
 
+# Interpreter sources may need special handling on some platforms
+set(interpreter_sources
+	BytecodeInterpreter.cpp
+	DebugBytecodeInterpreter.cpp
+	MHInterpreter.cpp
+)
+
+add_library(j9vm SHARED
+	${main_sources}
+	${interpreter_sources}
+)
+
+if(OMR_OS_AIX AND (OMR_TOOLCONFIG STREQUAL "xlc"))
+	set_source_files_properties(${main_sources} COMPILE_FLAGS "-O3")
+	set_source_files_properties(${interpreter_sources} COMPILE_FLAGS "-O2 -qdebug=lincomm:ptranl:tfbagg")
+endif()
```

Comment:
Are the linux on P flags being handled in a separate PR? https://github.com/eclipse/openj9/blob/d6314e979b0c20821c9150ad3a63b56a31396656/runtime/makelib/targets.mk.ftl#L554-L556